canonical-web-and-design/canonical.com
# Add redirect from /intellectual-property-rights-policy

## Done
Add redirect from /intellectual-property-rights-policy

## QA
- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8002/intellectual-property-rights-policy
- Which should redirect to https://ubuntu.com/legal/intellectual-property-policy

## Issue / Card
Fixes https://github.com/canonical-web-and-design/ubuntu.com/issues/8441
